### DISCUSSION TITLE
What happens to my repositories once the Pro membership expires?

### DISCUSSION THREAD

**Discussion by little-software-engineer** [community/community#129354] [Repositories]
### Select Topic Area

Question

### Body

Hi everyone,

I was wondering what will happen to my GitHub repositories when my student Pro membership expires. Do they get deleted or something else? Thank you in advance.

**Comment by cetvoropek** [comment]
When a GitHub Pro subscription expires, the account transitions back to the free tier. This change primarily impacts features and limits but does not affect the availability or integrity of the repositories. Public repositories remain public and fully accessible, ensuring continued visibility and collaboration. Private repositories created during the Pro period retain their privacy, but the number of collaborators may be restricted to the limits of the free plan, typically capped at three. Advanced features exclusive to GitHub Pro, such as enhanced code review tools, repository insights, and additional CI/CD minutes and storage, become unavailable. Users may experience limitations if their usage exceeds the free plan’s data transfer and storage limits, potentially leading to restricted repository actions until usage is reduced or additional resources are purchased. Pro-specific support and advanced security features, like code scanning and secret scanning, are also discontinued. Despite these changes, all existing repositories remain intact and functional, ensuring no loss of data or accessibility.
